src/codeu/chat/server/Controller.java

```diff
@@ -147,6 +154,51 @@ public Conversation newConversation(Uuid id, String title, Uuid owner, Time crea
     return conversation;
   }
 
+  // Search for a user in the database and create the User object in the return.
+  @Override
+  public User searchUserInDatabase(String username, String pswd){
+    StringBuilder stringBuilder = new StringBuilder();
+    Gson gson = new Gson();
+
+    // Open the database file.
+    try (BufferedReader br = new BufferedReader(new FileReader("../src/codeu/chat/databases/users"))) {
```

Comment:
I've considered it and it would be a nice implementation to have a static class to save all the paths of databases.